src/range-input/range-input.ts

```diff
@@ -87,6 +90,10 @@ export class NgAisRangeInput extends BaseWidget {
     return 1 / Math.pow(10, precision);
   }
 
+  get canRefine() {
+    return this.state.range && this.state.range.min !== this.state.range.max;
```

Comment:
According to [the typings](https://github.com/algolia/angular-instantsearch/blob/380cf9bcd175922da41293a2d2c9d450188dd7ad/src/range-input/range-input.ts#L10), `this.state.range` is always defined. Is the first check necessary?

Reply:
solved here 89e1113